source/renderer/sidebar/store.js

```diff
@@ -11,9 +11,9 @@
  * END HEADER
  */
 
-const Vue = require('vue')
-const objectToArray = require('../../source/common/util/object-to-array')
-const findObject = require('../../source/common/util/find-object')
+const Vue = require('vue').default
```

Comment:
```suggestion
import Vue from 'vue'
```

I think, these two are equivalent.

Reply:
It's probably best to do this as part of the typescript migration https://github.com/Zettlr/Zettlr/issues/1186 (for which imports are the preferred way)